README.md

```diff
@@ -6,46 +6,24 @@
 
 As of August 14, 2020, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) is now the maintainer of the EPUB 3 specifications.
 
-## EPUB 3.3
+## Current version of EPUB: EPUB 3.3
 
-Editor's Drafts of documents under active development:
+On the 25th of May, 2023, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) has published three 
```

Comment:
Do we need "has"?

Reply:
Hm. I leave this question to you native Anglo-Saxon guys 😀. What they taught me at school is that past perfect is (also) used when the (not too distant) past actions have a tangible influence/presence on the present. But there may be niceties that evade me...

Reply:
"had published" would be past perfect; "has published" is present tense. By dating the action in the past but making the tense in the present, it does sound weird. You could say "As of the 25th... the working group has" if the idea is we plan to publish more, but dropping "has" probably makes the most sense.

```suggestion
On the 25th of May, 2023, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) published three 
```

Reply:
👍